app/src/androidTest/java/org/mozilla/fenix/ui/robots/SearchRobot.kt

```diff
@@ -311,13 +268,41 @@ class SearchRobot {
         pasteText.click()
     }
 
+    fun clickSearchEnginePrompt(rule: ComposeTestRule, searchEngineName: String) =
+        rule.onNodeWithText("Search $searchEngineName").performClick()
+
     fun expandSearchSuggestionsList() {
         onView(allOf(withId(R.id.search_wrapper))).perform(
             closeSoftKeyboard(),
         )
         awesomeBar.swipeUp(2)
     }
 
+    fun verifyTranslatedFocusedNavigationToolbar(toolbarHintString: String) =
+        assertTranslatedFocusedNavigationToolbar(toolbarHintString)
+
+    fun verifySearchEngineShortcuts(rule: ComposeTestRule, vararg searchEngines: String) {
+        mDevice.findObject(
+            UiSelector().resourceId("$packageName:id/awesome_bar"),
+        ).swipeUp(1)
```

Comment:
Is this brittle?

Reply:
No, from my point of view.
It's being used 3x in two tests which were run 100x on Firebase.

I there were any problems I think it would have failed at some point.